lib/server/services/lhcPeriod/LhcPeriodStatisticsService.js

```diff
@@ -46,6 +47,21 @@ class LhcPeriodStatisticsService {
         return data ? lhcPeriodStatisticsAdapter.toEntity(data) : null;
     }
 
+    /**
+     * Find a lhc period model by its name or id including its statistics
```

Comment:
I think here you should really state `Find an LHC period statistics model` which is really different than `Find an LHC period model with statistics`
This applies to the whole file

Reply:
updated